node-tests/blueprints/eslint-config-simplabs-test.js

```diff
@@ -38,8 +38,7 @@ describe('eslint-config-simplabs blueprint', function() {
       .then(() => emberGenerate(['eslint-config-simplabs']))
       .then(() => {
         expect(file('.eslintrc.js')).to.contain('extends: \'simplabs\'');
-        expect(file('tests/.eslintrc.js'))
-          .to.equal(`module.exports = {\n  extends: 'simplabs/configs/ember-qunit',\n};\n`);
+        expect(file('tests/.eslintrc.js')).to.equal(file(`${__dirname}/fixtures/.eslintrc.qunit.js`));
```

Comment:
Wouldn't a template string be better as we'd have the expected content right here instead of having to look it up in a separate file?